README.md

```diff
@@ -27,6 +38,32 @@ Using lazy.nvim:
 }
 ```
 
+For roslyn
+
+```lua
+{
+  "iabdelkareem/csharp.nvim",
+  dependencies = {
+    "williamboman/mason.nvim", -- Required, automatically installs omnisharp
```

Comment:
Is it required for roslyn mode?

Reply:
Right, so I think you should remove mason from this example to keep it minimal.

Reply:
Already done in this commit https://github.com/iabdelkareem/csharp.nvim/pull/11/commits/684ee9cfbcbf26abe23dfe02f76ac6ea6429e1ca